app/components/status_page_component.rb

```diff
@@ -14,8 +16,8 @@ class StatusPageComponent < BaseComponent
   attr_reader :status, :icon
 
   def initialize(status: :error, icon: nil)
-    if !ICONS.key?(status)
-      raise ArgumentError, "`status` #{status} is invalid, expected one of #{ICONS.keys}"
+    if !VALID_STATUS.include?(status)
```

Comment:
[comment] prefer `if` with a positive test, or consider `unless` in a guard clause, e.g. `raise ... unless VALID_STATUS.include?(status)`